modules/geo-layers/package.json

```diff
@@ -45,6 +45,6 @@
     "@deck.gl/core": "^8.0.0",
     "@deck.gl/layers": "^8.0.0",
     "@deck.gl/mesh-layers": "^8.0.0",
-    "@loaders.gl/core": "^2.2.0"
+    "@loaders.gl/core": "^2.2.3"
```

Comment:
Don’t bump `peerDependencies`. These do not affect package resolution and we intend to keep them relaxed.

Reply:
Ok, I think those were automatically incremented by `ocular-bump`..